portal/src/main/webapp/WEB-INF/jsp/dashboard/dashboard.jsp

```diff
@@ -344,7 +344,7 @@
                         document.title = _cancerStudy.name
                         $("#study_name").html(_cancerStudy.name);
                         var _desc = _cancerStudy.description;
-                        if(_cancerStudy.pmid !== null){
```

Comment:
should we also check if it's null?

Reply:
I think its not required as it depends on legacy api, but to be safe will include it.

Reply:
done!